styles.css

```diff
@@ -0,0 +1,100 @@
+body {
```

Comment:
most of the style here can be achieved with bootstrap 
example: p-*, border-*, d-flex